packages/next/src/client/components/react-dev-overlay/_experimental/internal/components/errors/dev-tools-indicator/dev-tools-indicator.tsx

```diff
@@ -267,7 +304,7 @@ function DevToolsPopover({
                   index={1}
                   label="Try Turbopack"
                   value={<ExternalIcon />}
-                  href="https://nextjs.org/docs/app/api-reference/turbopack"
+                  onClick={() => setIsTurbopackInfoOpen(true)}
```

Comment:
I think we can remove the externalIcon above since it's not opening an external link anymore.

Reply:
Sorry, ended up syncing with design on this: let's do this instead
![CleanShot 2025-02-18 at 12 27 45@2x](https://github.com/user-attachments/assets/287464d4-f766-4f7e-a360-f77b421417ac)
(so right arrow instead of "Enabled | Disabled")